provider-directory-nginx-proxy/documentation/production.clinical-fhir.metadata.json

```diff
@@ -0,0 +1,216 @@
+{
+  "id": "health-api-provider-directory-r4-capability",
```

Comment:
It's fine to bring these six files over as placeholders, but a lot of the internals don't make sense for PDAPI, e.g. we're not going to have `Patient` or `Observation` resources. For now the body can just be `{}`